install.sh

```diff
@@ -390,17 +390,17 @@ start_docker
 
 echo ""
 echo -e "\n🟡 Pulling the latest container images for Lightdash. To run as sudo it may ask for system password\n"
-sudo LIGHTDASH_INSTALL_ID="$INSTALLATION_ID" LIGHTDASH_INSTALL_TYPE="$LIGHTDASH_INSTALL_TYPE" docker-compose -f docker-compose.yml pull
+sudo LIGHTDASH_INSTALL_ID="$INSTALLATION_ID" LIGHTDASH_INSTALL_TYPE="$LIGHTDASH_INSTALL_TYPE" PGPASSWORD="password" LIGHTDASH_SECRET="not very secret" docker-compose -f docker-compose.yml pull
 
 echo ""
 echo "🟡 Starting the Lightdash containers. It may take a few minutes ..."
 echo
 # The docker-compose command does some nasty stuff for the `--detach` functionality. So we add a `|| true` so that the
 # script doesn't exit because this command looks like it failed to do it's thing.
 if [[ $setup_type == 'local_dbt' ]]; then
-    sudo LIGHTDASH_INSTALL_ID="$INSTALLATION_ID" LIGHTDASH_INSTALL_TYPE="$LIGHTDASH_INSTALL_TYPE" PORT="$port" DBT_PROJECT_DIR="$dbt_project_dir" docker-compose -f docker-compose.yml up --detach --remove-orphans || true
+    sudo LIGHTDASH_INSTALL_ID="$INSTALLATION_ID" LIGHTDASH_INSTALL_TYPE="$LIGHTDASH_INSTALL_TYPE" PORT="$port" PGPASSWORD="password" LIGHTDASH_SECRET="not very secret" DBT_PROJECT_DIR="$dbt_project_dir" docker-compose -f docker-compose.yml up --detach --remove-orphans || true
```

Comment:
Why inline this? it's much harder to find out what's going on when you're setting up

Reply:
I also found it to not be ideal but it was the only way that worked. 
I might have been doing something wrong so I'm open to suggestions.